README.rst

```diff
@@ -87,13 +87,16 @@ file of this repository, and can be defined in your project's ``conf.py`` via
 .. code:: python
 
     html_theme_options = {
+        # Base Options
         'typekit_id': '',
         'canonical_url': '',
         'analytics_id': '',
         'logo_only': False,
         'display_version': True,
         'prev_next_buttons_location': bottom,
         'style_external_links': False,
+         # CSS Options
+        'nav_search_background_color': #2980B9,
```

Comment:
I guess the `#` will need to be escaped somehow?

Reply:
I think it's just a string in the `conf.py` directory. Wouldn't it simply be quoted?